### DISCUSSION TITLE
New Features

### DISCUSSION THREAD

**Discussion by andrekane** [organicmaps/organicmaps#9322] [Ideas]
So I been planned these new features:
1. New UI for Main Screen and Car Navigatior
2. Improve Turn by Turn Directions
3. Add Specific Car Navigator Colors
4. Add Picture by Picture Mode
5. Add New Places
6. Add New Route Selector (Motorcycle, Truck, etc)
7. New OSM Data for September 2024
8. Add Traffic Lights in the Non-Car Navigator
9. Improved Direction Arrows in The Car Navigator, which they are one by one when completed
10. New Hub Option
11. Improve Fix Crash
12. New Map Styles (Default, Bright, Niifgaard, etc)
13. Improve Road Alerts 
14. New "News" Option, when you check for the new updates
15. Add Home, Work in the Main Screen
16. Add Ruler Option
17. Added Tram To The Public Transport 
18. Add Custom Themes (Blue, Green) 
19. Add App Color (Optional) 
20. Add Record Activity
21. Add Record Track
22. Add Quick Ruler
23. Add Lane Bubbles
24. Add Current Street Name In Car-Navigator
25. Add Highlighting The Religion Or Country (You can disable it on settings)
26. Add The Time Depending On The Day Or Time
27. Add Free Ride

**Comment by patepelo** [comment]
Some ideas are great, some are already implemented and others are already planned. I recommend you to open specific issues per topic. Otherwise theres no follow up to convert into actions.

**Comment by andrekane** [comment]
Soo, can you add them? It's okay to not add the others, label them to 1 to 27, so that I can know :)

**Reply by patepelo** [reply]
I propose you instead to focus and pick your favorite and realistic feature. Invest appropriate time into searching from the 2.5k issues, if you find it, add your collaborative ideas. If it's not there, proceed to open an issue and spark the discussions.
